pages/common.py

```diff
@@ -130,10 +130,32 @@ def body(content):
 <script src="assets/js/custom.js"></script>
 <!-- SLICK CAROUSEL -->
 <script type="text/javascript" src="assets/slick/slick.min.js"></script>
-
+<!-- BOOTBOX -->
+<script type="text/javascript" src="assets/js/bootbox.min.js"></script>
+
+<!-- Chat link modal alert handler, this is the only place it can go at the moment  -->
+<script type="text/javascript">
+jQuery(document).on("click", '[title="chatlink"]', function(e) {
+    bootbox.dialog({
+        message: "$chat_modal_message",
+        title: "$chat_modal_title",
+        buttons: {
+            main: {
+                label: "OK",
+                className: "btn-primary",
+                callback: function() {
+                    window.open("https://webchat.freenode.net/?randomnick=1&channels=freenet");
+                }
+            }
+        }
+    });
+});
+</script>
 </body>
 """
-    return string.Template(template).substitute(content=content)
+    chat_modal_title = _("Please note")
+    chat_modal_message = _("This chat is staffed by volunteers, and it may be that no one is around right now. Please ask your question, and someone will answer within several hours.")
```

Comment:
This contradicts the context in `help.py`, where the user is instructed to wait a few minutes instead of several hours. Either `help.py` or this text should be changed.